src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -29,6 +25,8 @@ protected override void ConnectHandler(UIButton nativeView)
 			nativeView.TouchUpOutside += OnButtonTouchUpOutside;
 			nativeView.TouchDown += OnButtonTouchDown;
 
+			SetupDefaults(nativeView);
```

Comment:
The defaults is setting static fields. Either the defaults should be instances on the handler, or it should do some checks to make sure it only reads it once. Otherwise this does work for no reason.